node/src/main.rs

```diff
@@ -318,6 +334,20 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
 
     debug!(logger, "Setting up Sentry");
 
+    // Obtain DISABLE_BLOCK_INGESTOR setting
+    let disable_block_ingestor: bool = matches
+        .value_of("disable-block-ingestor")
+        .unwrap()
+        .parse()
+        .expect("Invalid disable block ingestor value");
```

Comment:
And here. Also, I'd make this message
```
invalid --disable-block-ingestor/DISABLE_BLOCK_INGESTOR value
```
or something like that to tie it to the flag/var.